libdnf.spec

```diff
@@ -154,6 +155,7 @@ popd
 %{_libdir}/pkgconfig/%{name}.pc
 %{_includedir}/%{name}/
 %{_datadir}/gir-1.0/Dnf-*.gir
+%{_datadir}/vala/vapi/
```

Comment:
Yes, there are a few -vala subpackages, but the vast majority are putting the vala bindings directly in -devel. e.g. on my system:

$ rpm -qf /usr/share/vala/vapi/* | sort | uniq
caribou-devel-0.4.21-2.fc26.x86_64
colord-devel-1.3.4-1.fc26.x86_64
colord-gtk-devel-0.1.26-3.fc24.x86_64
dconf-devel-0.26.0-1.fc25.x86_64
evolution-data-server-devel-3.23.4-1.fc26.x86_64
folks-devel-0.11.3-4.fc26.x86_64
gcr-devel-3.20.0-3.fc26.x86_64
gedit-devel-3.22.0-3.fc26.x86_64
gegl03-devel-0.3.10-1.fc26.x86_64
gmime-devel-2.6.23-1.fc26.x86_64
gnome-online-accounts-devel-3.23.4-1.fc26.x86_64
grilo-devel-0.3.2-4.fc26.x86_64
gspell-devel-1.2.2-1.fc26.x86_64
gssdp-devel-1.0.1-1.fc26.x86_64
gtksourceview3-devel-3.23.1-1.fc26.x86_64
gtkspell3-devel-3.0.9-1.fc26.x86_64
gtk-vnc2-0.6.0-1.fc26.x86_64
gupnp-av-devel-0.12.10-1.fc26.x86_64
gupnp-devel-1.0.1-1.fc26.x86_64
gupnp-dlna-devel-0.10.5-2.fc26.x86_64
gvnc-0.6.0-1.fc26.x86_64
ibus-devel-1.5.14-5.fc26.x86_64
libatasmart-devel-0.19-9.fc24.x86_64
libcanberra-devel-0.30-12.fc26.x86_64
libchamplain-devel-0.12.14-2.fc26.x86_64
libgcab1-devel-0.7-1.fc25.x86_64
libgdata-devel-0.17.6-3.fc26.x86_64
libgee-devel-0.18.1-1.fc26.x86_64
libgexiv2-devel-0.10.4-3.fc26.x86_64
libgit2-glib-devel-0.24.4-3.fc26.x86_64
libgnome-games-support-devel-1.2.1-1.fc26.x86_64
libgnome-keyring-devel-3.12.0-7.fc26.x86_64
libgusb-devel-0.2.9-1.fc25.x86_64
libgweather-devel-3.20.4-1.fc26.x86_64
libmediaart-devel-1.9.0-5.fc26.x86_64
libosinfo-vala-1.0.0-1.fc26.x86_64
librsvg2-devel-2.40.16-2.fc26.x86_64
libsecret-devel-0.18.5-2.fc26.x86_64
libsoup-devel-2.57.1-1.fc26.x86_64
libvirt-gconfig-devel-1.0.0-1.fc26.x86_64
libvirt-glib-devel-1.0.0-1.fc26.x86_64
libvirt-gobject-devel-1.0.0-1.fc26.x86_64
NetworkManager-glib-devel-1.6.0-1.fc26.x86_64
NetworkManager-libnm-devel-1.6.0-1.fc26.x86_64
PackageKit-glib-devel-1.1.5-1.fc26.x86_64
pulseaudio-libs-devel-10.0-2.fc26.x86_64
spice-glib-devel-0.33-2.fc26.x86_64
spice-gtk3-vala-0.33-2.fc26.x86_64
telepathy-glib-vala-0.24.1-4.fc25.x86_64
tracker-devel-1.11.3-1.fc26.x86_64
uhttpmock-devel-0.5.0-2.fc24.x86_64
vte291-devel-0.46.1-1.fc26.x86_64